backend-api/template.yaml

```diff
@@ -315,7 +316,25 @@ Resources:
               Service: lambda.amazonaws.com
             Action: sts:AssumeRole
       Policies:
-        - PolicyName: AsyncTokenFunctionSsmPolicy
+        - PolicyName: DynamoReadSessionsTable
+          PolicyDocument:
+            Version: "2012-10-17"
+            Statement:
+              - Effect: Allow
+                Action:
+                  - dynamodb:GetItem
+                  - dynamodb:Query
```

Comment:
We haven't actually tested this yet. What Dynamo commands do we run? I think it's just GetItem and PutItem from memory? 

Please may you check and update these - no need to test as we're blocked by KMS

Reply:
Updated